docs/examples/kitchen-sink/graphviz.rst

```diff
@@ -0,0 +1,28 @@
+========
+Graphviz
+========
+
+Inheritance Diagram
+-------------------
+
+Using `sphinx.ext.inheritance_diagram`, inheritance diagrams can be generated
+through `sphinx.ext.graphviz`.  If the output of the inheritance diagrams are
+in `svg` format, they can be made to conform to light or dark mode.
+
+To have the inheritance-diagram render to `svg`, inside `conf.py`, you need
```

Comment:
```suggestion
To have the inheritance-diagram render to SVG, inside ``conf.py``, you need
```

(nitpick) I'm 50/50 on formatting as `svg` rather than "SVG"... I waffle on whether the acronym for a file format is "code" (maybe it is when used as a file extension, but not when simply referenced as a format?)

Reply:
was never good at knowing when to use one back-tick or two back-ticks.  Fixed.